internal/conf/nacos.go

```diff
@@ -67,10 +77,12 @@ type timeout struct {
 }
 
 type Nacos struct {
-	Host    []string
-	Prefix  string
-	Weight  int
-	Timeout timeout
+	Host     []string
+	Prefix   string
+	User     string
+	Password string
```

Comment:
Different hosts must only use the same user name and password?

Reply:
In general the account and password within the cluster are the same, and it can be optimized if needed later